library/src/com/astuetz/PagerSlidingTabStrip.java

```diff
@@ -170,13 +173,11 @@ public PagerSlidingTabStrip(Context context, AttributeSet attrs, int defStyle) {
         underlineColor = textPrimaryColor;
         dividerColor = textPrimaryColor;
         indicatorColor = textPrimaryColor;
-        int paddingLeft = a.getDimensionPixelSize(PADDING_LEFT_INDEX, padding);
-        int paddingRight = a.getDimensionPixelSize(PADDING_RIGHT_INDEX, padding);
+        int padding = a.getDimensionPixelSize(PADDING_INDEX, 0);
+        paddingLeft = padding > 0 ? padding : a.getDimensionPixelSize(PADDING_LEFT_INDEX, 0);
+        paddingRight = padding > 0 ? padding : a.getDimensionPixelSize(PADDING_RIGHT_INDEX, 0);
```

Comment:
as per typical Android views, if `android:padding` is given, it will override `android:padding[Side]`